website/docs/advanced-concepts/granular-access-control/reference/default-roles.md

```diff
@@ -0,0 +1,670 @@
+---
+description: >-
+  Granular Access Control - Default Roles in Appsmith
+title: Default Roles
+hide_title: true
+---
+
+<!-- vale off -->
+
+<div className="tag-wrapper">
+ <h1>Default Roles </h1>
+
+<Tags
+tags={[
+{ name: "Business", link: "https://www.appsmith.com/pricing", additionalClass: "business" }
+]}
+/>
+
+</div>
+
+<!-- vale on -->
+This page provides detailed insights into the individual permissions associated with the default roles available in Appsmith.
+
+## Instance Administrator Role
+
+The Instance Administrator Role operates at the highest level within the platform, primarily focusing on tasks such as configuring instances, managing user groups and roles, creating workspaces, and monitoring audit logs. 
+
+### Permissions
+
+To perform different tasks, the Instance Administrator role in Appsmith has the following permissions:
+
+  <div className="gac-permissions">
+        <p className="permission-footnote">
+        <span style={{color: 'green', fontWeight: 'bold'}}>(✓)</span> Permission Assigned
+        </p>
+        <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+            <thead>
+                <tr>
+                    <th></th>
+                    <th>Create</th>
+                    <th>Edit</th>
+                    <th>Delete</th>
+                    <th>View</th>
+                    <th>Execute</th>
```

Comment:
This execute should not be there, there's no execute in the Groups & Roles tab of roles.

Reply:
removed the blank column